typescript/index.ts

```diff
@@ -1,4 +1 @@
-import { version, Parser } from "./pkg/cooklang_wasm";
-
-export { version, Parser };
-export type { ScaledRecipeWithReport } from "./pkg/cooklang_wasm";
+export * from "./pkg/cooklang_wasm";
```

Comment:
I am hesitant to do this. We have less control over the build artifacts of `wasm-pack`. If we export everything, technically anything changing from this file could possibly be a breaking change. By explicitly exporting certain functions and variables, we have tighter control over the API and types. This also makes it easier to document.